crates/red_knot_python_semantic/resources/mdtest/snapshots/unresolved-import.md_-_Unresolved_import_diagnostics_-_Using_`from`_with_a_resolvable_module_but_unresolvable_item.snap

```diff
@@ -0,0 +1,33 @@
+---
+source: crates/red_knot_test/src/lib.rs
+expression: snapshot
+---
+---
+mdtest name: unresolved-import.md - Unresolved import diagnostics - Using `from` with a resolvable module but unresolvable item
+mdtest path: crates/red_knot_python_semantic/resources/mdtest/diagnostics/unresolved-import.md
+---
+
+# Python source files
+
+## a.py
+
+```
+```
+
+## mdtest_snippet__1.py
+
+```
+1 | from a import does_not_exist  # error: [unresolved-import]
+```
+
+# Diagnostics
+
+```
+error: lint:unresolved-import
+ --> /src/mdtest_snippet__1.py:1:15
+  |
+1 | from a import does_not_exist  # error: [unresolved-import]
+  |               ^^^^^^^^^^^^^^ Module `a` has no member `does_not_exist`
```

Comment:
Can we extend this to include one existing symbol? Such that it would read:

```suggestion
1 | from a import does_exist, does_not_exist  # error: [unresolved-import]
  |                           ^^^^^^^^^^^^^^ Module `a` has no member `does_not_exist`
```

Reply:
Done.